### DISCUSSION TITLE
(help) the sound is not coming out on public page and widget

### DISCUSSION THREAD

**Discussion by Scryter** [AzuraCast/AzuraCast#3774] [Q&A] [answered]
The audio is not outputting on the audio player, both on the public page and on the widget. on the dashboard, where the radio list appears, on those blue buttons (marked in the image), I managed to get the "IP / radio / port" of each radio to listen to the radio, apparently it is normal there. so with this ip, i used an html5 (audio) player tag to embed each radio on each site. it is not ideal but it works, however, if you can help me, or make available a previous version of azuracast, thank you very much.

![aa](https://user-images.githubusercontent.com/60343264/106959020-e2fc6580-6718-11eb-9d49-f97d8f282341.PNG)

**Comment by freygabor** [comment, marked as answer]
- Go to settings, and tick the 2 options: "Prefer Browser URL (If Available)" and "Use Web Proxy for Radio".
- Restart your server
Will be fine.

I had a same issue and this was the problem :)

![Screenshot 2021-02-05 at 18 15 12](https://user-images.githubusercontent.com/78565352/107072715-4a301d80-67de-11eb-81bc-653249f443ba.png)

**Reply by Scryter** [reply]
Thank you so much. you saved my life.
